### PR TITLE
feat(octavia): adding loadbalancer overview page

### PR DESCRIPTION
**Pull request by JacquesLarique**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/octavia-batch-1`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-10680
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (n/a)

## Description

Adding loadbalancer overview page with modal to edit the name

### CODE REVIEW

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    this.trackCancel();
+    this.goBack();
+  }
+
+  update() {
+    this.trackConfirm();
+    this.isLoading = true;
+    this.$http
+      .put(
+        `/cloud/project/${this.projectId}/region/${this.region}/loadbalancing/loadbalancer/${this.loadbalancer.id}`,
```
Comment (BrunoMarquesOVH) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js:24]: Hello,

Shouldn't this be put in the service for reuse purpose?

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/service.js
Comment (qpavy) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/service.js:1]: Don't forget to catch all the $http calls to display error with queryId with the Alerter

**Review by darsene — APPROVED** [review]

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.name = this.loadbalancer.name;
+  }
+
+  cancel() {
+    this.trackCancel();
+    this.goBack();
```
Comment (ghyenne) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js:16]:
```suggestion
    return this.goBack();
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.routing.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            name: `${trackingSuffix}-success`,
+            type: 'action',
+          });
+        },
+        trackFailure: /* @ngInject */ (atInternet, trackingSuffix) => () => {
+          atInternet.trackClick({
+            name: `${trackingSuffix}-error`,
+            type: 'action',
+          });
+        },
```
Comment (ghyenne) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.routing.js:46]:
```suggestion
        trackClick: /* @ngInject */ (atInternet, trackingSuffix) => (hit) => atInternet.trackClick({
            name: `${trackingSuffix}${hit}`,
            type: 'action',
          }),
```

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/routing.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+            projectId,
+            region,
+            loadbalancer.vipNetworkId,
+            loadbalancer.vipSubnetId,
+          ),
+        openEditNameModal: /* @ngInject */ ($state) => () => {
+          $state.go(
+            'octavia-load-balancer.loadbalancer.general-information.edit-name',
+          );
+        },
```
Comment (ghyenne) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/routing.js:51]:
```suggestion
        goToEditNameModal: /* @ngInject */ ($state) => () => $state.go(
            'octavia-load-balancer.loadbalancer.general-information.edit-name',
          ),
```
As we're changing state, I think the name should be changed.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+  "octavia_load_balancer_overview_config_provisioning_status": "Statut de provisioning",
+  "octavia_load_balancer_overview_config_provisioning_status_active": "actif",
+  "octavia_load_balancer_overview_config_provisioning_status_creating": "en cours de création",
+  "octavia_load_balancer_overview_config_provisioning_status_deleted": "supprimé",
+  "octavia_load_balancer_overview_config_provisioning_status_deleting": "en cours de suppression",
+  "octavia_load_balancer_overview_config_provisioning_status_error": "erreur",
+  "octavia_load_balancer_overview_config_provisioning_status_updating": "en cours de mise à jour",
+  "octavia_load_balancer_overview_config_operating_status": "Statut d'opération",
+  "octavia_load_balancer_overview_config_operating_status_degraded": "dégradé",
+  "octavia_load_balancer_overview_config_operating_status_draining": "dégradé",
```
Comment (ghyenne) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json:17]: The same content ?

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  "octavia_load_balancer_overview_config_operating_status_offline": "offline",
+  "octavia_load_balancer_overview_config_size": "Taille",
+  "octavia_load_balancer_overview_config_size_small": "S",
+  "octavia_load_balancer_overview_config_size_medium": "M",
+  "octavia_load_balancer_overview_config_size_large": "L",
+  "octavia_load_balancer_overview_config_private_network": "Réseau privé",
+  "octavia_load_balancer_overview_config_subnet": "Subnet",
+  "octavia_load_balancer_overview_config_private_ip": "Adresse IP privée",
+  "octavia_load_balancer_edit_name_success": "Le nom de votre Load Balancer a bien été modifé.",
+  "octavia_load_balancer_edit_name_error": ""
```
Comment (ghyenne) [inline packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json:30]:
```suggestion
```
Remove it or fill in.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json (hunk 5 above)
Comment (JacquesLarique) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/translations/Messages_fr_FR.json:17]: Strange but yes

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/routing.js (hunk 4 above)
Comment (JacquesLarique) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/routing.js:51]: Not sure about that, indeed we change state, but we display it in a modal (as a child state).
Anyway this name does not seem correct, I think we should either use **openEditNameModal** or **goToEditName**.
I would be interested to know what the team think.

File: packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js (hunk 2 above)
Comment (JacquesLarique) [reply packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/overview/edit-name/edit-name.controller.js:16]: I don't see the purpose of returning here: cancel is used in the template where return value is ignored

**Review by qpavy — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**